router/worker.go

```diff
@@ -411,6 +411,11 @@ func (w *worker) processDestinationJobs() {
 					if err != nil {
 						errorAt = routerutils.ERROR_AT_TF
 						respStatusCode, respBody := types.RouterUnMarshalErrorCode, fmt.Errorf("transformer response unmarshal error: %w", err).Error()
+						jobIds := lo.Map(destinationJob.JobMetadataArray, func(jobMetadata types.JobMetadataT, _ int) int64 {
+							return jobMetadata.JobID
+						})
+						// limiting the log to print 10KB of transformed payload
+						w.logger.Errorw("transformer response unmarshal error for message: %s, jobs: %v", "message", string(destinationJob.Message[:10*1024]), "jobIDs", jobIds)
```

Comment:
It will throw `slice bounds out of range out` if it is less than 10KB. 
```suggestion
                         truncatedMessage := misc.TruncateStr(string(destinationJob.Message), int(10*bytesize.KB))
						w.logger.Errorw("transformer response unmarshal error for message: %s, jobs: %v", "message", truncatedPayload, "jobIDs", jobIds)
```